### ISSUE TITLE
Support cutting elements from the slice with append

### ISSUE THREAD

**Issue by AlexVanin** [nspcc-dev/neo-go#999]
Sometimes you need to remove element from the slice and one of the convenient ways to that is to use `append`. 

```go
package foo
func Main() int {
	a := []int{0,1,2,3,4,5}
	a = append(a[:2], a[3:]...)
	return len(a)
}
```

This code does not compile in neo-go with `(SUBSTR): can't convert Array to ByteArray` message You have to create new slice, which is totally fine, but may be a bit confusing. So it would be nice to support this feature if possible.

**Comment by fyrchik** [comment]
This happens because subslicing is supported only for `[]byte` via `SUBSTR` instruction.
There is no similar instruction for regular `Array`s and the only way to support this is via loops (which can be done manually anyway).

However for your specific case, there is a `REMOVE` instruction, which can be wrapped in separate builtin in `2.x` and possibly invoked directly after #941 .

**Comment by fyrchik** [comment]
Closed via #1021.
Your example:
```golang
package foo
import "github.com/nspcc-dev/neo-go/pkg/interop/util"
func Main() int {
	a := []int{0,1,2,3,4,5}
	util.Remove(a, 2)
	return len(a)
}
```
